package-lock.json

```diff
@@ -0,0 +1,138 @@
+{
+  "name": "platform",
+  "lockfileVersion": 2,
+  "requires": true,
```

Comment:
if dependencies changes, it will be important to keep the `package-lock.json` up to date too